tensorboard/pip_package/build_pip_package.sh

```diff
@@ -159,26 +166,32 @@ from tensorboard.plugins.beholder import Beholder, BeholderHook
 tb.notebook.start  # don't invoke; just check existence
 "
 
-  # Exhaustively test various sequences of importing tf.summary.
-  test_tf_summary() {
-    # First argument is subpath to test, e.g. '' or '.compat.v2'.
-    import_attr="import tensorflow as tf; a = tf${1}.summary; a.write; a.scalar"
-    import_as="import tensorflow${1}.summary as b; b.write; b.scalar"
-    import_from="from tensorflow${1} import summary as c; c.write; c.scalar"
-    printf '%s\n' "${import_attr}" "${import_as}" "${import_from}" | python -
-    printf '%s\n' "${import_attr}" "${import_from}" "${import_as}" | python -
-    printf '%s\n' "${import_as}" "${import_attr}" "${import_from}" | python -
-    printf '%s\n' "${import_as}" "${import_from}" "${import_attr}" | python -
-    printf '%s\n' "${import_from}" "${import_attr}" "${import_as}" | python -
-    printf '%s\n' "${import_from}" "${import_as}" "${import_attr}" | python -
-  }
-  test_tf_summary '.compat.v2'
-  is_tf_2() {
-    python -c "import tensorflow as tf; assert tf.__version__[:2] == '2.'" \
-      >/dev/null 2>&1
+  # Check if we are testing with or without tf
+  is_tf() {
+    python -c "import tensorflow as tf" >/dev/null 2>&1
   }
-  if is_tf_2 ; then
-    test_tf_summary ''
+  if is_tf ; then
```

Comment:
Is there a reason that this isn’t just `if [ -n "${smoke_tf}" ]`? We
need the active check for TF 2.0 because we can’t easily discern the
TensorFlow major version from the Pip distribution specification, but
just checking whether we installed TensorFlow at all seems easy enough.

Reply:
Good point - changing.